src/main/java/in/oogway/ologger/networkLogger.java

```diff
@@ -0,0 +1,64 @@
+package in.oogway.ologger;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class networkLogger implements oogwayLogger {
+    static Logger networkLogger = LoggerFactory.getLogger(networkLogger.class);
+    Level level;
+
+    public networkLogger(){
+        this.level = Level.WARN;
+    }
+
+    @Override
+    public void setLevel(Level level) {
+        switch (level)
+         {
+             case CRITICAL :this.level=level;
```

Comment:
Rather than a switch case. You should use map based approach.

```java
HashMap<String, Level> m = new Hashmap
This.level = M.get(levelArg)
```

However in your case you are setting what you get in argument. So you can literally not do any of this and just set this.level = LevelArg